src/js/injected_methods.js

```diff
@@ -49,6 +61,16 @@ DataView.prototype.readU32 = function() {
     }
 };
 
+DataView.prototype.readU64 = function() {
+    if (this.byteLength >= this.offset+8) {
+        const right = this.readU32();  //64bit not working properly in js, have to split into two 32bit
```

Comment:
Why is this not working? This only happens in Betaflight or there is some info out about this?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER

I don't wanted to use BigInt because I would have to duplicate some functions then.

Reply:
Ok, it's simply the size of the data, not any kind of problem or bug. I don't know if have a "false" function for that is good, or simply call twice the readU32 directly when reading the MSP command as we do for other similar data (I think remember the flight modes maybe?). I think is more clear. @haslinghuis what do you think?

Reply:
Agree with using a 2 x 32 bit struct to complement the 32 and 64 led_strip.

Reply:
> Agree with using a 2 x 32 bit struct to complement the ledstrip_32 and ledstrip_64 feature.

So is it good now or should I change it?
I don't understand exactly.

Reply:
Please use 32 bit data structure. With ledstrip 64 enabled we can use a second 32 bit structure.
Hardest part is remaining backwards compatibility with previous versions.

Reply:
> Please use 32 bit data structure. With ledstrip 64 enabled we can use a second 32 bit structure. Hardest part is remaining backwards compatibility with previous versions.

I think you misunderstand the problem. It's not related to led strip length (32 or 64), instead it's the led config length, each configured led has this which contains it's configuration. Previously it was 32bit long but with the new feature I couldn't fit into this so I had to expand it to 64 bit.

The problem is that JavaScript can't use 64 bit numbers only 53 bit so I call the the readU32 twice and return them in an array. I added readU64 where this is happening. It's for clearity and code readability.
I think @McGiverGim wants me not to implement this readU64 and use twice readU32 directly in the code.

I already fixed the backward compatible problem.

Reply:
Agree with @McGiverGim Please use twice U32

Reply:
Alright, I change it.
Limon added the 64 led feature to cloud build.